test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -13872,10 +13874,11 @@ class Reservation(ResourceResv):
 
     dflt_attributes = {}
 
-    def __init__(self, username=None, attrs={}):
+    def __init__(self, username=None, attrs={}, hosts=[]):
```

Comment:
It's dangerous to use mutable types as default arguments.  It means every call to this function will use the same memory.  If the list is modified in one call, it will still be modified in subsequent call.  It's safer to set it to None and then say if hosts is None: hosts = [].  I see that attrs in the same way, but it's also wrong.  Fix that one as well?

Reply:
Both the attrs and hosts fixed.